src/rust/lib/graph-editor/src/component/edge.rs

```diff
@@ -13,7 +13,10 @@ use ensogl::display::scene::Scene;
 use ensogl::display::shape::*;
 use ensogl::display::traits::*;
 use ensogl::display;
+use ensogl::gui::component::ShapeViewEvents;
 use ensogl::gui::component;
+use nalgebra::UnitComplex;
```

Comment:
Lets use `rotate_90(v:Vector2<f32>) -> Vector2<f32> { Vector2(v.y,v.x) }`, or something like that instead :)

Reply:
I moves to use `Rotate2` which is more appropriate. I would like to keep  the genral rotation to keep the code more robust against new shapes that might have a diffeent rotation.